model/cassandra/registry.yaml

```diff
@@ -0,0 +1,75 @@
+groups:
+  - id: registry.cassandra
+    type: attribute_group
+    display_name: Cassandra Attributes
+    brief: >
+      This group defines attributes for Cassandra.
+    attributes:
+      - id: cassandra.coordinator.dc
+        type: string
+        stability: development
+        brief: >
+          The data center of the coordinating node for a query.
+        examples: "us-west-2"
+      - id: cassandra.coordinator.id
+        type: string
+        stability: development
+        brief: >
+          The ID of the coordinating node for a query.
+        examples: "be13faa2-8574-4d71-926d-27f16cf8a7af"
+      - id: cassandra.consistency.level
+        brief: >
+          The consistency level of the query. Based on consistency values from [CQL](https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/dml/dmlConfigConsistency.html).
+        type:
+          members:
+            - id: all
+              value: "all"
+              stability: development
+            - id: each_quorum
+              value: "each_quorum"
+              stability: development
+            - id: quorum
+              value: "quorum"
+              stability: development
+            - id: local_quorum
+              value: "local_quorum"
+              stability: development
+            - id: one
+              value: "one"
+              stability: development
+            - id: two
+              value: "two"
+              stability: development
+            - id: three
+              value: "three"
+              stability: development
+            - id: local_one
+              value: "local_one"
+              stability: development
+            - id: any
+              value: "any"
+              stability: development
+            - id: serial
+              value: "serial"
+              stability: development
+            - id: local_serial
+              value: "local_serial"
+              stability: development
+        stability: development
+      - id: cassandra.idempotence
```

Comment:
doesn't follow the [`{domain}.{thing}.{a property of the thing}`](https://github.com/open-telemetry/semantic-conventions/issues/1581#issuecomment-2486925892) pattern (I don't have any better idea here, just commenting in case you have thoughts)

Reply:
thanks! changed to `cassandra.query.is_idempotent`

Reply:
i'm slightly anti on using the `is_` prefix, wdyt?

Reply:
but can totally live with it if we apply consistently (or have reasons for when we apply it and when we don't)

Reply:
would you prefer `cassandra.query.idempotent` ?

Reply:
yeah